src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -74,6 +74,7 @@ public function __construct(
     /**
      * @Transform /^product "([^"]+)"$/
      * @Transform /^"([^"]+)" product$/
+     * @Transform :product
```

Comment:
We should use `:productName` in step definitions if we want to get product name. E.g.
```gherkin
I create :productName # I want to get product name as a string
I add :product to the cart # I want to get product entity
```